trulens_eval/examples/experimental/streamlit_appui_example.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.        initial_app_loader=load_langchain_app
</pre><p>Maybe add the comment about needing blank memory as the reason here in the notebook so people setting it up know why its done this way.</p><p>Also; is memory.clear() something that can be utilized instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Added notes to explain why that is insufficient.